tests/security/lynis/lynis_setup.pm

```diff
@@ -32,10 +32,8 @@ sub run {
 
     select_console "root-console";
 
-    if (is_sle) {
-        add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1);
-        zypper_call("in lynis", timeout => 300);
-    }
+    add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1) if is_sle;
+    zypper_call("in lynis", timeout => 300);
 
     # Record the pkgs' version for reference
     my $results = script_output("rpm -qi lynis");
```

Comment:
> rpm -q --qf '%{version}\n' lynis (this cmd can show the pkg version, anyway, I am not asking for you to revise the code, just learnt from Ben)

I know this way but my way is good too since there are no further usage of the output. Thanks for the reminder.